src/Extension/Converter/ESLintViolationReportConverter.php

```diff
@@ -108,10 +108,13 @@ protected function getPackages(string $projectDirectory, array $files): array
      */
     protected function mapSeverity(int $severity): string
     {
-        return match ($severity) {
-            1 => ViolationInterface::SEVERITY_WARNING,
-            2 => ViolationInterface::SEVERITY_ERROR,
-            default => '',
-        };
+        switch ($severity) {
```

Comment:
`return [1 => ViolationInterface::SEVERITY_WARNING, 2 => ViolationInterface::SEVERITY_ERROR][$severity] ?? ViolationInterface::SEVERITY_INFO;`
but I thought it was fixed 
![image](https://user-images.githubusercontent.com/5063777/182356006-1d192942-232a-4f79-b423-1959d5de3a3f.png)

Reply:
Maybe this part was lost after merging
Will fix

Reply:
Fixed